doc/src/fpga_dfl_drv/fpga_dfl_drv.md

```diff
@@ -25,40 +27,40 @@ https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/drivers/fp
+|----------------------|---------------------------------------------------|-------------------------------------------------------|-------------------------------------------------------|
+| hello_fpga           | YES                                               | YES                                                   | YES                                                   |
+| fpgaconf             | YES                                               | YES                                                   | NO                                                    |
+| fpgad                | YES                                               | NO                                                    | NO                                                    |
+| fpgainfo             | YES                                               | YES                                                   | NO                                                    |
+| fpgametrics          | YES                                               | YES                                                   | NO                                                    |
+| hello_events         | YES                                               | NO                                                    | NO                                                    |
+| hssi_config          | YES                                               | NO                                                    | NO                                                    |
+| hssi_loopback        | YES                                               | NO                                                    | NO                                                    |
+| object_api           | YES                                               | YES                                                   | NO                                                    |
+| mmlink               | YES                                               | YES                                                   | NO                                                    |
+| bist_app             | YES                                               | NO                                                    | NO                                                    |
+| coreidle             | YES                                               | NO                                                    | NO                                                    |
+| discover_fpgas       | YES                                               | NO                                                    | NO                                                    |
+| fpga_dma_test        | YES                                               | NO                                                    | YES                                                   |
```

Comment:
fpga_dma_test is unsupported in patchset2 but is supported in patchset1?

Reply:
would like to know..